DeAceroDLabs/marketplace-frontend
# Feature/dmp 737 Comma as thousand separator

## What is done

[[DMP-737] Comma as thousand separator](https://deacero.atlassian.net/browse/DMP-737)

## How is done

- Added  the  built-in toLocaleString() method to use comma as thousand separator.
- The toLocaleString() converts number to string, that is why I use it only when the price is finally displayed.

## How to test

1. Add a product that price is over thousand to cart.
2. Verify that on every view price is displayed as wanted.

## Evidence

![image](https://user-images.githubusercontent.com/81311646/208489698-4ca33603-c039-408b-83ee-adcf8e72c4d0.png)
![image](https://user-images.githubusercontent.com/81311646/208489937-dfb306f0-3be4-4855-92dd-72bd58a08066.png)

![image](https://user-images.githubusercontent.com/81311646/208490016-5457b6df-94d6-43f3-99b3-1ae3401ae910.png)
![image](https://user-images.githubusercontent.com/81311646/208490083-dd96da4e-ef72-429a-8a27-3a55bfc33e9e.png)
